diffpy/snmf/factorizers.py

```diff
@@ -0,0 +1,17 @@
+import numpy as np
+import scipy.optimize
+def lsqnonneg(C,d):
```

Comment:
use full words for variables to capture their meaning (in the context of the function)

Reply:
Fixing